apps/src/templates/SchoolAutocompleteDropdown.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React, {Component} from 'react';
-import VirtualizedSelect from 'react-virtualized-select';
+import loadable from 'loadable-components';
+const VirtualizedSelect = loadable(() => import('react-virtualized-select'));
 import 'react-virtualized/styles.css';
 import 'react-select/dist/react-select.css';
 import 'react-virtualized-select/styles.css';
```

Comment:
does this also need to be lazy-loaded?

Reply:
might as well! The stylesheets total 15KB. I've added a wrapper so that they get lazy-loaded too.

Reply:
these are now getting extracted into the dynamically loaded module:
![Screen Shot 2019-11-06 at 10 16 25 AM](https://user-images.githubusercontent.com/8001765/68325635-adcd1780-007e-11ea-9366-f8c37224a2f3.png)